config/.jshint.json

```diff
@@ -53,7 +53,7 @@
     "punycode",
     "qq",
     "blueimp",
-
+    "L",
```

Comment:
We have 3 blocks here:

1. defined by our gems
2. for the jasmine tests
3. our own global variables

Please put the new variable in the first block (ordered alphabetically) and keep the empty line.